roles/openshift_logging/templates/elasticsearch.yml.j2

```diff
@@ -32,7 +32,7 @@ discovery:
 
 gateway:
   expected_master_nodes: ${NODE_QUORUM}
```

Comment:
And since we currently only support master nodes serving data, we should probably drop this entirely, and just use the `expected_nodes` parameter.